tests/FakeItEasy.IntegrationTests/FakeItEasy.IntegrationTests.csproj

```diff
@@ -39,7 +39,7 @@
     <ProjectReference Include="..\..\src\FakeItEasy\FakeItEasy.csproj" />
     <ProjectReference Include="..\..\tests\FakeItEasy.Tests\FakeItEasy.Tests.csproj" />
     <ProjectReference Include="..\FakeItEasy.IntegrationTests.External\FakeItEasy.IntegrationTests.External.csproj">
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <Private>false</Private>

```

Comment:
There's a story here... The reason for this was to force `FakeItEasy.IntegrationTests` to depend on `FakeItEasy.IntegrationTests.External` so that `FakeItEasy.IntegrationTests.External` is always built before `FakeItEasy.IntegrationTests`. This was initially done with a dependency in the solution file. However, because of https://github.com/dotnet/cli/issues/6258, running `dotnet restore` at the solution level didn't restore packages for `FakeItEasy.IntegrationTests`. My first attempt to fix it was to remove the dependency from the solution, and to reference the project but without actually referencing the assembly. This worked fine on the command line, but sometimes failed to build in Visual Studio (in fact, it *always* failed... it had seemed to work at some point, but no longer). But in fact, it doesn't matter if we reference it in the project: since we don't actually use it, there's no reference in the generated assembly. The `<Private>false</Private>` just prevents it from being copied to the output directory.